infrastructure/src/main/java/org/apache/geode/infrastructure/aws/LaunchCluster.java

```diff
@@ -397,11 +397,17 @@ private static Image getNewestImage() {
     DateTimeFormatter inputFormatter =
         DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss.SSS'Z'", Locale.ENGLISH);
 
+    String purpose = System.getProperty("PURPOSE");
```

Comment:
Properties also supports a default value. So you can drop the empty check and do this.

`String purpose = System.getProperty("PURPOSE", "geode-benchmarks")`